committing.rst

```diff
@@ -347,6 +337,8 @@ will still be issued but the changeset will be applied anyway::
 Clones Setup
 ------------
 
+// TODO is that still needed?
```

Comment:
Do we still recommend having multiple clones? It is worth talking about forks and working with them, so that changes land in people's private forks rather than in the main repo. But given the fact that git branches are so easy do we recommend having multiple dirs, one per each branch. I do believe some of core-devs prefer that approach, but is it worth having it in here, still?